.rubocop.yml

```diff
@@ -81,6 +88,12 @@ RSpec/DescribeClass:
   Exclude:
     - spec/project/**/*.rb
 
+RSpec/MultipleExpectations:
+  Max: 2
```

Comment:
Why?

Reply:
A normal unit test has only one expectation,  but a cop test case is special and has one of the following three patterns:

1. `expect_no_offenses(...)`
2. `expect_offense(...)`
3. `expect_offense(...); expect_correction(...)`

To allow for the 3rd pattern, I set this `Max: 2`.

Reply:
[RSpec style guide's guideline regarding single expectation per example](﻿﻿﻿https://rspec.rubystyle.guide/#expectation-per-example) is less strict.
Since this is not directly related to introducing this new cop, I suggest extracting this into a separate PR. And I'm still uncertain if we'd really like to constrain ourselves to the ~one~ two expectations style.

Reply:
I think that the point is:
This PR introduces a cop that would complain about our own specs
So it also adds our expectations helper to the Language configuration
Previously MultipleExpectations would count expectations per example as 0, as it was not aware of those helpers. 
Now that it’s aware, it’s triggering an offence 
Hence also the tweak in configuration

Reply:
Makes total sense, thanks.